src/main/java/fr/insee/publicenemy/api/configuration/auth/AuthorityRole.java

```diff
@@ -0,0 +1,11 @@
+package fr.insee.publicenemy.api.configuration.auth;
```

Comment:
maybe not needed. You can just use hasRole on @PreAuthorize annotations. If you don't have a lot of endpoints, juste configure them in OidcSecurityConfiguration